src/main/data/facilities.json

```
@@ -5112,6 +5112,16 @@
     "code": "PSC",
     "description": "POSTCOLOR"
   },
+  {
+    "code": "PSH",
+    "contact": {
+      "address": "60 Cole Valley Road, Birmingham UK, B28 0DF",
```

```suggestion
      "address": "60 Cole Valley Road, Birmingham, B28 0DF, UK",
```

https://www.postoffice.co.uk/mail/how-to-address-mail